### ISSUE TITLE
Channel feature

### ISSUE THREAD

**Issue by shahzadthathal** [johnnymast/mysql_websocket_chat#27]
Hi,

Is it possible to create a channel for 3,4 user so they can talk to each other and also chat privately?

Currently, if I send a message it delivers to all user but I want a group chat for different users so each user have their group of 3,4 people and they can chat in that channel or privately.

Thanks

**Comment by johnnymast** [comment]
yes for sure that would be possible its not in there right now but it could be added easy.
